buildpack/detect.go

```diff
@@ -54,12 +54,12 @@ func (b *Descriptor) Detect(config *DetectConfig) DetectRun {
 	defer os.RemoveAll(planDir)
 
 	planPath := filepath.Join(planDir, "plan.toml")
-	if err := ioutil.WriteFile(planPath, nil, 0777); err != nil {
+	if err := ioutil.WriteFile(planPath, nil, 0600); err != nil {
```

Comment:
Not 100% sure on this change, but the acceptance tests are passing, so I think it's fine?

Reply:
I don't think we need to execute `plan.toml` - so this feels right